pkgs/top-level/python-packages.nix

```diff
@@ -10501,6 +10501,8 @@ self: super: with self; {
 
   pystac = callPackage ../development/python-modules/pystac { };
 
+  pystac-client = callPackage ../development/python-modules/pystac-client { };
```

Comment:
Should this package be in `pkgs/by-name`? I'm not sure what the guideline is for sub-packages, but this file calls at least one by-name package.

Reply:
The python package must be added to python-packages.nix to be called correctly.